frontend/src/app/login-form/login-form.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="login-container">
+  <mat-card class="login-card">
+    <mat-card-header class="login-header">
+      <div mat-card-avatar class="login-card-icon"></div>
+      <mat-card-title>Sign In</mat-card-title>
+    </mat-card-header>
+    <mat-card-content class="login-card-content">
+      <mat-form-field appearance="fill">
+        <mat-label>Username</mat-label>
+        <input matInput type="text" value="ala" #username />
```

Comment:
Try to use `placeholder` property in input instead of `mat-label`. You may like the effect ;)

Reply:
I added placeholder and leave `mat-label` also. Looks good to me ;)